importApp/src/main/java/com/commercetools/dataimport/joyrideavailability/AvailabilityJobConfiguration.java

```diff
@@ -59,7 +59,7 @@
     private static final Logger logger = LoggerFactory.getLogger(AvailabilityJobConfiguration.class);
     private int productsImportStepChunkSize = 1;
     private final List<String> channelsIds = new ArrayList<>();
```

Comment:
do you really need this list if you already save all channels in the list below?